legion/engine/core/assets/assets.inl

```diff
@@ -543,7 +543,7 @@ namespace legion::core::assets
     template<typename AssetType>
     inline L_ALWAYS_INLINE void asset<AssetType>::destroy()
     {
-        AssetCache<AssetType>::destoy(m_id);
+        AssetCache<AssetType>::destroy(m_id);
```

Comment:
can we put this on a separate branch and get this merged into develop immediately?